.github/workflows/dockerByPR.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker CI By PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/**'
```

Comment:
we should only rebuild on "code changes" - pretty sure we can exclude .github/** and some other directories

Reply:
Change added